abel/tools/center.py

```diff
@@ -223,6 +250,41 @@ def find_center_by_center_of_mass(IM, verbose=False, round_output=False,
     return center
 
 
+def find_center_by_convolution(IM, **kwargs):
+    """ Center the image by convolution of two projections along each axis.
+
+        code from the ``linbasex`` juptyer notebook
+
+    Parameter
+    -------
+    IM: numpy 2D array
+        image data
+    Returns
+    -------
+    center: tuple
+        (row-center, col-center)
+
+    """
+    # projections along axis=0 of image (rows)
```

Comment:
I tried to leave a message here before, but maybe it got lost. Basically, np.sum(IM, axis=0) takes the sum along axis 0, so it's confusing to describe it as the "projection along axis=0", since this leave it ambiguous if the projection runs along axis=0 or if it's been collapsed along axis=0.